openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -2913,7 +2913,7 @@ struct AMDGPUPluginTy final : public GenericPluginTy {
   /// Create an AMDGPU plugin and initialize the AMDGPU driver.
   AMDGPUPluginTy()
       : GenericPluginTy(getTripleArch()), Initialized(false),
-        HostDevice(nullptr) {}
+        HostDevice(nullptr), IsAPU(false), OMPX_HSAXnack("HSA_XNACK", false) {}
```

Comment:
(I wrote this before I saw Shilei's comment)
I understand you followed existing code but I think we should (maybe in a pre-commit) move these initializations to the member definition. It's cumbersome to have them here, in order, as one needs to look at 2 places to understand type and initial value.

Reply:
actually, the existing code had the initialization in the correct place. This is plainly my mistake for this patch.